src/spouts/readHygiene.py

```diff
@@ -0,0 +1,32 @@
+from collections import Counter
+from pymongo import MongoClient
+from streamparse import Spout
+import json
+
+class readHygiene(Spout):
+    outputs = ['rec']
```

Comment:
Lets call it hygiene_record instead of rec to add more context globally in the file